### PR TITLE
Import wordpress blog to eleventy markdown

### PR DESCRIPTION
**Pull request by Thingee**
The initial import of the Ceph blog to eleventy. Fixes some syntax issues from the import as well.

### CODE REVIEW

File: src/en/news/blog/2008/blog/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (Pete-Robelou) [inline src/en/news/blog/2008/blog/index.md:1]: Is this post needed?

The content describes why the blog was setup.

File: src/en/news/blog/2008/client-cache-coherency/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Client cache coherency"
+date: "2008-03-24"
+author: "sage"
+tags:
```
Comment (Pete-Robelou) [inline src/en/news/blog/2008/client-cache-coherency/index.md:5]: I've noticed lots of articles don't have any tags, is that correct?

File: src/en/news/blog/2008/delayed-capability-release/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: "Delayed capability release"
+date: "2008-04-18"
+author: "sage"
```
Comment (Pete-Robelou) [inline src/en/news/blog/2008/delayed-capability-release/index.md:4]: Would it be possible to update to the full name and capitalise the author for all the articles?

File: src/en/news/blog/2011/hiring/index.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Hiring!"
```
Comment (Pete-Robelou) [inline src/en/news/blog/2011/hiring/index.md:2]: Is this article still needed ?

File: src/en/news/blog/2013/a-gentle-introduction-to-the-erasure-coding/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+tags: 
+---
+
+![](images/erasure-coding-intro.jpg "A gentle introduction to the erasure coding")
+
+The erasure coding is currently a very hot topic for distributed storage systems. It has been part of the Ceph roadmap for almost a year and Swift guys recently brought the discussion to the table. Both of them have planned to implement the erase code functionality so I thought it might be interesting to give a high level overview about the erasure coding principles. Before we start, I’d like to point out that **I don’t take any credit for this article**. I just read the wonderful white paper [“Erasure Coding vs. Replication: A Quantitative Comparison”](http://www.cs.rice.edu/Conferences/IPTPS02/170.pdf) written by Hakim Weatherspoon and John D. Kubiatowicz from Computer Science Division University of California, Berkeley. Many thanks to them. While ready the paper, I found their introduction to the erasure code easy to understand for a novice like me :).
+
+  
+
+# I. Introduction
```
Comment (Pete-Robelou) [inline src/en/news/blog/2013/a-gentle-introduction-to-the-erasure-coding/index.md:14]: I appreciate it's unrealistic to go through all the old articles and update the headings so they are semantic but this article has multple `h1` tags.

**Review by Pete-Robelou — COMMENTED** [review]
I found this `/2015/8741/` which only contains images so it isn't being included in the Blog. Could it be removed?

I noticed there are lots of articles from 2016 and onwards that only contain images so none of them are being included in the Blog. Is this an error and they should have an accompanying markdown file or could they be removed? There are too many for me to list here.

There is an article in `/2020/12619/` and it's title is `Ceph Community Newsletter, August 2020 Edition`, could the folder name be updated to reflect the title so it's consistent with the other articles?

File: src/en/news/blog/2011/hiring/index.md (hunk 4 above)
Comment (liewegas) [reply src/en/news/blog/2011/hiring/index.md:2]: i think it makes sense to keep these for historical completeness.

File: src/en/news/blog/2008/blog/index.md (hunk 1 above)
Comment (liewegas) [reply src/en/news/blog/2008/blog/index.md:1]: as the first blog post, i think so!

File: src/en/news/blog/2013/a-gentle-introduction-to-the-erasure-coding/index.md (hunk 5 above)
Comment (Thingee) [reply src/en/news/blog/2013/a-gentle-introduction-to-the-erasure-coding/index.md:14]: Updated.

File: src/en/news/blog/2008/client-cache-coherency/index.md (hunk 2 above)
Comment (Thingee) [reply src/en/news/blog/2008/client-cache-coherency/index.md:5]: Correct, some posts were never given one.

File: src/en/news/blog/2008/delayed-capability-release/index.md (hunk 3 above)
Comment (liewegas) [reply src/en/news/blog/2008/delayed-capability-release/index.md:4]: We can clean these up in a follow-on PR?

Comment (adamduncan) [reply src/en/news/blog/2008/delayed-capability-release/index.md:4]: Yep, happy to go with usernames for launch. We'll leave to a future PR to revisit 👍

**Comment by adamduncan** [comment]
@Thingee, I've removed the remaining posts with no markdown files (https://github.com/ceph/ceph.io/pull/174/commits/594794b528fc1a8a06181aca277d04d8490de7cf). I presume a lot of these were the old Planet articles which have been removed.

This takes care of 261 unused images. As images are all processed and optimised at build-time, so this greatly improves the build time and CI's efforts.

**Review by adamduncan — APPROVED** [review]

**Pull request merged**